R/states.R

```diff
@@ -14,12 +14,13 @@ create_states <- function(psq) {
   S <- sum(psq$S_0)
   E <- sum(psq$E1_0, psq$E2_0)
   IMild <- sum(psq$IMild_0)
+  IAsymp <- sum(psq$IAsymp_0)
   ICase <- sum(psq$ICase1_0, psq$ICase2_0)
   IOxGetLive <- sum(psq$IOxGetLive1_0, psq$IOxGetLive2_0)
   IOxGetDie <- sum(psq$IOxGetDie1_0, psq$IOxGetDie2_0)
   IOxNotGetLive <- sum(psq$IOxNotGetLive1_0, psq$IOxNotGetLive2_0)
   IOxNotGetDie <- sum(psq$IOxNotGetDie1_0, psq$IOxNotGetDie2_0)
-  IMVGetLive <- sum(psq$IMVGetLive1_0, psq$IMVGetLive2_0)
+  IMVGetLive <- sum(psq$IMVGetLive1_0, psq$pIMVGetLive2_0)
```

Comment:
Just noticed this typo: 

```suggestion
  IMVGetLive <- sum(psq$IMVGetLive1_0, psq$IMVGetLive2_0)
```